style.css

```diff
@@ -125,3 +161,196 @@ button {
 .banner-scroll-btn button img {
   padding-top: 2px;
 }
+
+/* end hero section */
+
+/* work section */
+.works-section-title {
+  margin-top: 0;
+}
+
+.cards-container {
+  display: grid;
+  gap: 24px;
+  grid-template-columns: 1;
+}
+
+.work-card {
+  display: flex;
+  flex-direction: column;
+  background: var(--neutral-color);
+  border: 1px solid var(--secondary-color-50);
+  border-radius: 8px;
+  overflow: hidden;
+}
+
+.work-card__head {
+  height: 200px;
+}
+
+.work-card__body {
+  color: var(--secondary-color);
+  background: #fff;
+  border-radius: 8px 8px 0 0;
+}
+
+.card-title {
+  font-weight: 600;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  text-align: center;
+  margin: 1rem;
+}
+
+.card-btns {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card-btn {
+  font-weight: 600;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  padding: 8px 12px;
+  background: var(--secondary-color-50);
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+}
+
+.cta-btn {
+  text-align: center;
+}
+
+.cta-btn a,
+.form-submit-btn {
+  display: inline-block;
+  color: #fff;
+  font-size: 1rem;
+  padding: 12px;
+  text-align: center;
+  margin: 24px auto;
+  border-radius: 4px;
+  background: var(--highlight-color);
+}
+
```

Comment:
Kindly implement these design changes for the works section of your project so youthe  have a better UI and a good attention to details : 

- [ ] the heading for the section `My recent Works` have a line under it in the deisgn given please implement this change for your design 😄 

Figma design :
![image](https://github.com/thasipablo/Personal-Portfolio/assets/20537712/29cb71d2-15d9-402c-bbaf-358074fd7f83)

Your design :
![image](https://github.com/thasipablo/Personal-Portfolio/assets/20537712/cc565787-6dfe-4e82-82ce-fee37e9f5ae7)